_posts/2018-03-16-first-steps-into-gen-servers.md

```diff
@@ -0,0 +1,77 @@
+---
+layout: post
+title:  "Automating Processes with Gen Servers: A Beginner's Approach"
+date:   2018-03-16 10:00:00 -0400
+categories: elixir automation microservices
+author: Kelvin Ma
+---
+As a financial services company, Forward Financing invests a signficant amount of time and resources around timebound reporting -- think daily, weekly, monthly, etc., report generation. As opportunities for automation are concerned, it doesn't get more obvious than this.
```

Comment:
I think 'invests ... time and resources IN timebound reporting' sounds a little cleaner to me.

Reply:
👍